### PR TITLE
Make sure build is setup according to Django's tutorial

### PR DESCRIPTION
**Pull request by FinalAngel**
- use subfolder while `manage.py` stays in root
- ensure requirements are pulling the latest master

### CODE REVIEW

File: src/settings.py [hunk 1]
```diff
@@ -82,12 +83,18 @@
     'aldryn_bootstrap3',
     'aldryn_newsblog',
     'aldryn_style',
-    # 'aldryn_faq', // disabled as of https://goo.gl/zVBDTE
+    'aldryn_faq',
     'aldryn_events',
     'aldryn_jobs',
+    # aldryn_forms
```
Comment (vxsx) [inline src/settings.py]: ?

**Comment by vxsx** [comment]
overall LGTM

**Comment by vxsx** [comment]
(but build is failing)

**Comment by FinalAngel** [comment]
first one passed though, need it in for deploy...

**Pull request merged**